addons/base_vat/models/base_vat.py

```diff
@@ -73,14 +52,16 @@
     'sk': 'SK0012345675',
 }
 
-class res_partner(osv.osv):
+
+class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     def _split_vat(self, vat):
         vat_country, vat_number = vat[:2].lower(), vat[2:].replace(' ', '')
         return vat_country, vat_number
 
-    def simple_vat_check(self, cr, uid, country_code, vat_number, context=None):
+    @api.model
```

Comment:
@hmo-odoo 
Yes , it required because when call from https://github.com/odoo/odoo/blob/master/addons/website_sale/controllers/main.py#L548
It give error.